lib/cndb/omf.h

```diff
@@ -83,15 +83,14 @@ OMF_SETGET(struct cndb_ver_omf, cnver_captgt, 64);
 
 struct cndb_kvs_add_omf {
     struct cndb_hdr_omf hdr;
-    uint32_t            kvs_add_fanout;
+    uint32_t            kvs_add_pad2;
```

Comment:
Just delete the member.  No need to maintain compatibility with earlier 3.0.0 dev versions.

Reply:
Trying to preserve 8-byte alignment.  I could delete both pad members, but then we wouldn't have any reserve.
But I can reorg and move pad to below (uint64_t pad).

Reply:
i'll just delete it...

Reply:
Due to our OMF versioning we shouldn't need to reserve fields any of our OMF.  Is the benefit that we can sneak in changes in a compatible manner?  Even if we do that, we'd still want to bump the version in order prevent old code from reading new OMF.  I don't entirely mind if you leave it, but I don't see much of an advantage.

Reply:
Removed.

Reply:
I'm most concerned about alignement, but it's usually far less painful to put a "reserved" field into use than to grow a structure when the time arrives...

Reply:
Seems like you should keep it if it will be padded regardless since we do this in other parts of our OMF.

Reply:
The padding in general is only to maintain 8-byte alignment, although the pad field do also serve as "reserved" fields should we need them down the line...
And I think we have a few explicitly reserved fields here and there in some structs that are likely to need them...

Reply:
Which is to say, I would prefer we leave at least one reserved field in most structs for easy future expansion, as long as they don't eat up a lot of storage...

Reply:
I guess I don't see how it's easier since we use strict OMF versioning.

Here's how we would expand with the reserved field:
- Bump the version
- Rename the field
- Use it.

Here's how you do it w/o a reserved field:
- Bump the version
- Add the field
- Use it.

Reply:
WIth a reserved field initialized to zero you might not have to bump the version, you can just start using it with zero meaning "disabled", for example.  This can be used to avoid all that versioned structure churn that is a nightmare to generate, maintain, and pore through...